community/joe/checksums

```diff
@@ -0,0 +1 @@
+495a0a61f26404070fe8a719d80406dc7f337623788e445b92a9f6de512ab9de  joe-4.6.tar.gz
```

Comment:
What version of `kiss` are you using?

Reply:
The kiss community version I believe.

Reply:
Strange -- `kiss` hasn't put the name of the checksummed file after the checksum for quite some time. What do `kiss v` and `kiss l kiss` report? You're using either a very outdated `kiss`, or something has gone awry. 
When I run `kiss c` on your package, I just get `495a0a61f26404070fe8a719d80406dc7f337623788e445b92a9f6de512ab9de`.